src/motorError.cpp

```diff
@@ -0,0 +1,84 @@
+#include "motorError.h"
+
+Timeout evaluationTimeout;
+Timeout resetTimeout;
+
+InterruptIn motorErrorPin(D0);
+
+#define SEQUENCE_DELAY 1000ms
+#define RESET_DELAY 3000ms
```

Comment:
Consider moving these into the header and brief comment about what each value is doing in case someone wants to tune